### ISSUE TITLE
A design for : code.lesintegristes.net

### ISSUE THREAD

**Issue by gregoiredierendonck** [lesintegristes/lesintegristes-theme#41]

**Comment by bpierre** [comment]
Yes, we need an evolutive CSS framework to build any kind of demo (and the homepage will be built on it).

**Comment by bpierre** [comment]
A typical demo page should display:
- The demo title
- A link to the related article (important)
- A link to the code.lesintegristes homepage
- A link to the blog homepage
- A link to the code.lesintegristes repository on GitHub (“Fork this demo on GitHub”)

On the blog, I think we should display links to code.lesintegristes.net with a special style: maybe as a link with a picto, or a big button.
